doc/CHANGELOG.rst

```diff
@@ -7,9 +7,13 @@ DAGMC Changelog
 Next version
 ====================
 
+v3.2.3
```

Comment:
Something looks wrong here relative to the released version v3.2.3 - maybe needs a rebase?